game/static/game/css/landing.css

```
@@ -155,15 +155,16 @@ body {
     font-size: 0.95rem;
     font-weight: 500;
     transition: color 0.3s ease;
+    padding-bottom: 6px;
 }
 
 .nav-links a::after {
     content: "";
     position: absolute;
```

Stylelint (17.13.0)

[error] Duplicate property "position" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)